### ISSUE TITLE
add support of sklearn metrics

### ISSUE THREAD

**Issue by yurakuratov** [deepmipt/DeepPavlov#1446]
Want to contribute to DeepPavlov? Please read the [contributing guideline](http://docs.deeppavlov.ai/en/master/devguides/contribution_guide.html) first.

**What problem are we trying to solve?**:

Add out of the box support of sklearn metrics. Currently, to use metrics from sklearn user has to wrap them with @register_metric in custom py-files. 

**How can we solve it?**:

Sklearn metrics could be used further like:
```
{
        "name": "sklearn.metrics:precision_score",
        "inputs": [
          "y_onehot",
          "y_pred_probas"
        ]
}
```
**Are there other issues that block this solution?**:
```
Not that I am aware of
```

**Comment by Hyperion101010** [comment]
I would like to give it a try

**Comment by vaskonov** [comment]
Hey @Hyperion101010 let me know if you are still interested in contribution to DeepPavlov.

**Comment by IgnatovFedor** [comment]
Closed due to #1546
